lib/base.php

```diff
@@ -623,16 +623,32 @@ public static function init() {
 			throw new \RuntimeException('Could not set timezone to UTC');
 		}
 
+
 		//try to configure php to enable big file uploads.
-		//this doesn´t work always depending on the web server and php configuration.
-		//Let´s try to overwrite some defaults anyway
+		//this doesn´t work always depending on the webserver and php configuration.
+		//Let´s try to overwrite some defaults if they are smaller than 1 hour
+		// One hour is 3600 seconds
+		$time_limit = 3600;
+
+		$max_execution_time_from_ini = @ini_get('max_execution_time');
+		$biggest_max_execution_time = $time_limit;
+		if (isset($max_execution_time_from_ini)) {
+			$biggest_max_execution_time = max($time_limit, intval($max_execution_time_from_ini));
+		}
+		@ini_set('max_execution_time', strval($biggest_max_execution_time));
```

Comment:
Seems a bit complicated, would a simpler version work?

```suggestion
	if (intval(@ini_get('max_execution_time')?? 0) < 3600) {
		@ini_set('max_execution_time', strval(3600));
	}
```

Reply:
Well, i don't really appreciate @ in the code...but it's my own opinion.
Better to check the function exists before calling it instead.

Reply:
I have implemented the simpler solution now :)